RHEL6_7/properties.ini

```diff
@@ -1,3 +1,4 @@
 [preupgrade-assistant-modules]
 src_major_version=6
 dst_major_version=7
+ast_fmt_version=0.1
```

Comment:
`ast` is really bad abbreviation IMO. Come up with a name that quickly gives you the idea what it stores. I couldn't figure out what the `ast` could mean.
I vote for `assessment_format_version` or (if you like shorter names; P.S. I don't) `assessment_fmt_ver`.